networkmanager.te

```diff
@@ -363,6 +363,10 @@ optional_policy(`
 	vpn_relabelfrom_tun_socket(NetworkManager_t)
 ')
 
+optional_policy(`
+	openvswitch_stream_connect(NetworkManager_t)
```

Comment:
This interface does not exist. If you add it, the block should be after the openvpn block above.